src/main/java/com/conveyal/datatools/editor/models/transit/Stop.java

```diff
@@ -116,11 +116,13 @@ public static Stop fromJson(@JsonProperty("lat") double lat, @JsonProperty("lon"
         ret.stop_desc = stopDesc;
         ret.stop_lat = location.getY();
         ret.stop_lon = location.getX();
+        // TODO gtfs-lib value needs to be int
+//        ret.wheelchair_boarding = wheelchairBoarding.toGtfs();
```

Comment:
Ah, the GTFS-lib type is string. Yeah we should change that (in fact there's a comment in GTFS Lib to that effect) but for the time being should we just cast the int to a string?